### ISSUE TITLE
Use case: Author needs to download a file containing their paper's author list

### ISSUE THREAD

**Issue by drphilmarshall** [LSSTDESC/desc-publications#47]
Imagine: I am writing a paper and want to compile the PDF from latex files on my local computer. I do not want to manually keep the paper's author list up to date, so instead I download a small file from the Pub System and insert it into my paper.

Because I used the [`start_paper`](https://github.com/LSSTDESC/start_paper) tool to start the paper, I have a `Makefile` that calls [`mkauthlist`](https://github.com/kadrlica/mkauthlist) to generate a latex author list out of a file called [`authors.csv`](https://github.com/LSSTDESC/start_paper/blob/master/authors.csv). This tool could be used by the Pub System itself; that way, I could download the latex author list (`authors.tex`) directly, but I would need to specify the journal/latex class when generating that latex file.

**Comment by sethdigel** [comment]
Yes, we'll be well served by having the Pub Management System be able to run mkauthlist to generate the contents of authors.tex and/or the authors.csv file.  This typically will not need to be done very often for any given paper, possibly only once.  As noted in #84, for this to work usefully, the Lead authors will need to be able to specify the order of authors in the Pub Management System.
